suite-native/config/src/supportedNetworks.ts

```diff
@@ -1,9 +1,8 @@
-import { networks, NetworkSymbol, getMainnets, getTestnets } from '@suite-common/wallet-config';
+import { Network, NetworkSymbol, getMainnets, getTestnets } from '@suite-common/wallet-config';
 
-const deprecatedNetworks = ['dash', 'btg', 'dgb', 'nmc', 'vtc'];
+const discoveryBlacklist: NetworkSymbol[] = ['sol', 'dsol'];
 
-// ordering defined by product - it is done according to the data about the coins and usage of them in Desktop Suite
-export const mainnetsOrder: NetworkSymbol[] = [
+export const supportedMainnetNetworks: NetworkSymbol[] = [
```

Comment:
Since this is only for portfolio tracker I think we should reflect that in name, same for testnets

Reply:
[d322446](https://github.com/trezor/trezor-suite/pull/10241/commits/d3224461c37cbe1e69ccf60b6b83ee37767c1c1b)